pkgs/development/python-modules/python-logstash/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "python-logstash";
+  version = "0.4.6";
+
+  disabled = isPyPy;
```

Comment:
Do you set this for a reason?

Reply:
it was in the example.. and i am not sure how to test if pypy is working

Reply:
No need to test. Just don't set it without a reason.